packages/core/src/utils/hash.ts

```diff
@@ -15,7 +15,7 @@ export function createHashCounter(prefix = '', count = 0, offset = 10, msb = 35,
     count++;
 
     // Skip "ad" due to ad-blockers
-    if (virtualCount === 373) {
+    if (virtualCount === 373 || skipList.findIndex(s => s === virtualCount) > -1) {
```

Comment:
Why wouldn't this work?

```suggestion
    if (virtualCount === 373 || skipList.includes(virtualCount)) {
```

Reply:
It will probably work just fine. `findIndex` returns on first occurance so it will only search the list once. When searching for performance benchmarks for includes vs findIndex it looks like they're very similar in operations/s. But I'm fine with changing - I mean it looks nicer with includes anyways 😄

Reply:
I tested changing now and it requires some changes to the tsconfig as `includes` is not a part of es6.

Reply:
Well, no. `.indexOf()` and `.includes()` are more "similar" in performance. But not `.findIndex()`. It's even worth changing to `skipList.indexOf(virtualCount) > -1`.

![image](https://github.com/user-attachments/assets/2d0150c8-47dc-4a02-b557-004fe8b71ec0)

```suggestion
    if (virtualCount === 373 || skipList.indexOf(virtualCount) > -1) {
```

Reply:
Ah, damn. The benchmark I ran listed `.findIndex` as the result of `.indexOf`.